roofit/roofitcore/inc/RooDataHist.h

```diff
@@ -278,7 +279,7 @@ class RooDataHist : public RooAbsData, public RooDirItem {
   mutable std::vector<double> _maskedWeights; //! Copy of _wgt, but masked events have a weight of zero.
   mutable std::vector<double> _maskedSumw2; //! Copy of _sumW2, but masked events have a weight of zero.
  
-  mutable unsigned long _curIndex{std::numeric_limits<std::size_t>::max()}; // Current index
+  mutable ULong64_t _curIndex{std::numeric_limits<ULong64_t>::max()}; // Current index
```

Comment:
Since this change a persistent member, this class version number needs to be increased (and any derived classes).

Reply:
We have already increased the class version in this release cycle when I changed from `std::size_t` to `unsigned long`. So I think I don't need to do it again?

Reply:
Yes.  There might be user that used the master to produce some (private) data.  And unless you are planning many further changes, it will not affect any significant negative effects [eg. the default containers for schema has 16 slots]

Reply:
Ok!

Reply:
There are no derived classes of RooDataHist, by the way.